cmake/FindFuzzufCC.cmake

```
@@ -0,0 +1,14 @@
+if(NOT FUZZUF_CC_ROOT)
+  find_program( FUZZUF_CC_AFL_CC fuzzuf-afl-cc )
+  find_program( FUZZUF_CC_AFL_CXX fuzzuf-afl-cc++ )
+  find_program( FUZZUF_CC_IJON_CC fuzzuf-ijon-cc )
+else()
+  find_program( FUZZUF_CC_AFL_CC fuzzuf-afl-cc NO_DEFAULT_PATH PATHS ${FUZZUF_CC_ROOT}/ ${FUZZUF_CC_ROOT}/bin/ )
+  find_program( FUZZUF_CC_AFL_CXX fuzzuf-afl-c++ NO_DEFAULT_PATH PATHS ${FUZZUF_CC_ROOT}/ ${FUZZUF_CC_ROOT}/bin/ )
```

Due to this typo, `FUZZUF_CC_AFL_CXX` becomes NOT_FOUND, thus IJON PUTs are not going to be compiled and their tests are not run (though `fuzzuf-afl-cc++` is not actually required here)
```suggestion
  find_program( FUZZUF_CC_AFL_CXX fuzzuf-afl-cc++ NO_DEFAULT_PATH PATHS ${FUZZUF_CC_ROOT}/ ${FUZZUF_CC_ROOT}/bin/ )
```

---

fixed